src/shared/install.c

```diff
@@ -2609,34 +2609,21 @@ int unit_file_add_dependency(
                                      SEARCH_FOLLOW_CONFIG_SYMLINKS, changes, n_changes);
 }
 
-int unit_file_enable(
-                UnitFileScope scope,
-                UnitFileFlags file_flags,
-                const char *root_dir,
+static int _unit_file_enable(
```

Comment:
i think the underscore prefix would be fine if we have functions which are wrapper by something else and that something else is really the only way to use the function. But that isn't the case here, it's used by multiple callers.

maybe rename this to unit_file_enable_now()? or so? or prefix with do_?